atomic_reactor/util.py

```diff
@@ -685,8 +689,6 @@ def get_manifest(
                 logger.debug("skipping version %s due to status code %s",
                              version, ex.response.status_code)
                 return None, saved_not_found
-            else:
```

Comment:
is it safe to remove it?

Reply:
we shouldn't remove this just in case

Reply:
remnant of trial and error, added it back